libsoftwarecontainer/src/gateway/dbus/dbusgateway.h

```diff
@@ -82,6 +85,12 @@ class DBusGateway :
     virtual bool teardownGateway() override;
 
 private:
+
+    // JSON stuff. These are members to allow continous appends to the config
+    json_t *m_totConfig;
```

Comment:
can you find better name for this variable i don't think its explicit enough. Suggestion : entireConfig